.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 env:
   bashPass: \033[32;1mPASSED -
-  bashWarn: \033[33;1mWARNING -
+  bashInfo: \033[33;1mWINFO -
```

Comment:
Ridiculous Nit-pick: I would prefer 'info' logs to be a different color than yellow (maybe [blue](https://gist.github.com/vratiu/9780109#file-bash_aliases-L16)?). I think 'warn' being yellow makes the most sense. 

Obviously not important enough to block

Reply:
That makes sense, it's something we'd probably want to change in all the repos that do this to be consistent though.